parsec/runtime.h

```diff
@@ -286,6 +286,33 @@ int parsec_context_add_taskpool( parsec_context_t* context, parsec_taskpool_t* t
  */
 int parsec_context_remove_taskpool( parsec_taskpool_t* tp );
 
+/**
+ * Query PaRSEC context capabilities.
+ */
+
+typedef enum parsec_context_query_cmd_e {
+    PARSEC_CONTEXT_QUERY_NODES,
+    PARSEC_CONTEXT_QUERY_RANK,
+    PARSEC_CONTEXT_QUERY_DEVICES,
+    PARSEC_CONTEXT_QUERY_CORES,
+    PARSEC_CONTEXT_QUERY_ACTIVE_TASKPOOLS
+} parsec_context_query_cmd_t;
```

Comment:
Some documentation for vararg parameters (`PARSEC_CONTEXT_QUERY_DEVICES`) would be good.